src/main/java/com/google/opensesame/servlets/MentorObject.java

```diff
@@ -1,19 +1,43 @@
 package com.google.opensesame.servlets;
 
-public class MentorObject {
-  private String name;
-  private String description;
+import java.util.ArrayList;
 
-  public MentorObject(String nameInput, String descriptionInput) {
-    name = nameInput;
-    description = descriptionInput;
+public class MentorObject extends PersonObject {
+  private ArrayList<String> projectIDs;
+
+  public MentorObject(String nameInput, String gitHubIDInput) {
+    this(nameInput, gitHubIDInput, "", new ArrayList<String>(), new ArrayList<String>());
+  }
+
+  public MentorObject(String nameInput, String gitHubIDInput, String descriptionInput) {
+    this(
+        nameInput,
+        gitHubIDInput,
+        descriptionInput,
+        new ArrayList<String>(),
+        new ArrayList<String>());
+  }
+
+  public MentorObject(
+      String nameInput, String gitHubIDInput, String descriptionInput, ArrayList<String> tagInput) {
+    this(nameInput, gitHubIDInput, descriptionInput, tagInput, new ArrayList<String>());
+  }
+
+  public MentorObject(
+      String nameInput,
+      String gitHubIDInput,
+      String descriptionInput,
+      ArrayList<String> tagInput,
+      ArrayList<String> projectIDInputs) {
+    super(nameInput, gitHubIDInput, descriptionInput, tagInput);
+    projectIDs = projectIDInputs;
   }
 
-  public String getName() {
-    return name;
+  public ArrayList<String> getProjects() {
+    return projectIDs;
```

Comment:
This is super nit but I thought it was interesting. If the plan is to mutate the list through the functions provided like `addProject` below, it might be worth considering returning an immutable list here:
```java
return Collections.unmodifiableList(projectIDs);
```

Reply:
Ooh sounds cool!